heron/healthmgr/src/java/com/twitter/heron/healthmgr/diagnosers/SlowInstanceDiagnoser.java

```diff
@@ -76,6 +80,7 @@ public Diagnosis diagnose(List<Symptom> symptoms) {
       }
     }
 
+    LOG.info("result symptom: " + resultSymptom);
```

Comment:
Seems noisy and not very helpful during normal execution or when the problem is in other components.

Reply:
we found this function returned null. i think this log may help to find out why the detector failed to catch the abnormals.